src/main/java/controller/ListChats.java

```diff
@@ -88,11 +96,12 @@ public long getChatId() {
      * <li> LastSentMessageId </li>
      * </ol>
      */
-    ImmutableMap<String, Object> getChatInfoOfChatInMap(Chat chat, String usernameOfSecondUser) {
+    ImmutableMap<String, Object> getChatInfoOfChatInMap(Chat chat, Pair<String, String> secondUser) {
```

Comment:
Why are we using a Pair here?
Can these not be 2 arguments?

Reply:
We are using a pair at the place where we are calling this function, hence I used a Pair, should I change it to 2 arguments?